cms/templates/cms/program_page.html

```diff
@@ -5,6 +5,8 @@
 {% load wagtailimages_tags %}
 
 {% block title %}{% trans "MIT Micromasters" %}{% endblock %}
```

Comment:
This block is for the `<title>...</title>` element. It's probably a good idea to use `{{ page.program.title }}` there